libibverbs/verbs.h

```diff
@@ -517,6 +517,7 @@ enum ibv_wc_opcode {
 	IBV_WC_BIND_MW,
 	IBV_WC_LOCAL_INV,
 	IBV_WC_TSO,
+	IBV_WC_RDMA_FLUSH,
```

Comment:
Is it better and simpler to use IBV_WC_FLUSH here as kernel defines IB_UVERBS_WC_FLUSH?

Reply:
sounds good, updated